src/Cli/func/Actions/LocalActions/CreateFunctionAction.cs

```diff
@@ -309,6 +309,13 @@ public async Task UpdateLanguageAndRuntime()
                     throw new CliException("Selected language doesn't match worker set in local.settings.json." +
                         $"Selected worker is: {_workerRuntime} and selected language is: {workerRuntimeSelected}");
                 }
+
+                if (workerRuntimeSelected == WorkerRuntime.Java)
+                {
+                    ColoredConsole
+                        .WriteLine($"This action is not supported when using the core tools directly. Please use one of the supported environments to test {workerRuntimeSelected} apps locally as specified by" +
+                        $"https://learn.microsoft.com/en-us/azure/azure-functions/functions-develop-local?pivots=programming-language-java");
```

Comment:
placeholder - will prob add a fw link that points to https://learn.microsoft.com/en-us/azure/azure-functions/functions-develop-local?pivots=programming-language-java#local-development-environments